L1Trigger/L1TTrackMatch/python/L1TrackJetEmulationProducer_cfi.py

```diff
@@ -0,0 +1,65 @@
+import FWCore.ParameterSet.Config as cms
+from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
+
+L1TrackJetsEmulation = cms.EDProducer('L1TrackJetEmulatorProducer',
+	L1TrackInputTag= cms.InputTag("L1GTTInputProducer", "Level1TTTracksConverted"),
```

Comment:
These lines are not following the naming conventions. l1tTrackJetsEmulation, l1tGTTInputProducer

Reply:
The convention changed when I was working in the code... I have changed them now